### PR TITLE
Resume

### PR DESCRIPTION
**Pull request by mjrb**
current iteration:
- new `/resume` endpoint
- takes {email, token} via `POST`
- returns `{upload: "some url", download: "some url", exists: true|false}`

steps to completion:
- [x] deploy to dev
- [x] update documentation
- [x] review and merge
- [x] add prod bucket, iam user, and role
- [ ] deploy to prod

### CODE REVIEW

**Comment by mjrb** [comment]
sloves #62

File: resume.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    except ClientError as e:
+        return {"statusCode": 500, "body": "failed to connect to s3" + str(e)}
+
+def exists(email):
+    client = boto3.client("s3", **RESUME)
+    try:
+        info = client.head_object(Bucket=RESUME_BUCKET, Key=email + ".pdf")
+        return True
+    except ClientError as e:
+        if e.response["Error"]["Code"] != 404:
```
Comment (thekabistro) [inline resume.py]: != ?

Shouldn't it return a 404 if object not found?

Comment (mjrb) [reply resume.py]: correct. i also forgot to reraise so it returned none and thats why the test passed XD

**Comment by mjrb** [comment]
apparently pull request has different env vars than branch builds.

**Pull request merged**